server.js

```diff
@@ -1,16 +1,117 @@
 const express = require("express");
 const app = express();
 const port = 3000;
+const path = require('path');
 const studentRoutes = require("./src/students/routes");
+const bodyParser = require('body-parser');
+console.log("Created server");
+const html = require('html');
+const pug = require('pug');
 
-app.get("/" , (req,res) => {
+app.set('view engine', 'pug');
+app.get('/', (req, res) => {
+    // Render the Pug template
+    res.render('index', {
+      title: 'My Website',
+    });
+  });
+app.use(bodyParser.json());
+/*app.get("/" , (req,res) => {
     res.send("Hello World");
 });
+console.log("Created server");
+//app.use(express.json());
+app.use(express.static(path.join(__dirname, 'public')));
 
-app.use(express.json());
-
-app.use("/students", studentRoutes)
-
+app.get('/', async(req, res) => {
+    res.render('index.html')
+});
+//app.use("/students", studentRoutes)
+app.post('/submit', studentRoutes);
 app.listen(port, () => {
     console.log(`App is listening at port ${port}`)
+}); */
+const pg = require("pg")
+
+const pool = new pg.Pool({
```

Comment:
What if you want to configure this password, username as some file or SECRET? 

What is secure way to save configuration in project?